src/compiler/transformers/classFields.ts

```diff
@@ -245,16 +481,21 @@ namespace ts {
                     addRange(expressions, generateInitializedPropertyExpressions(staticProperties, temp));
                     expressions.push(startOnNewLine(temp));
 
-                    pendingExpressions = savedPendingExpressions;
                     return inlineExpressions(expressions);
                 }
             }
 
-            pendingExpressions = savedPendingExpressions;
             return classExpression;
         }
 
         function transformClassMembers(node: ClassDeclaration | ClassExpression, isDerivedClass: boolean) {
+            // Declare private names.
+            node.members.forEach(member => {

```

Comment:
Use `for..of` instead of `forEach` here to avoid the extra function allocation and call stack overhead.